### ISSUE TITLE
Feature Request: add "Created On" category to one time passwords management

### ISSUE THREAD

**Issue by sunjam** [nextcloud/server#10560]
![created-on-one-time-passwords](https://user-images.githubusercontent.com/1787238/43740250-8fedb010-997f-11e8-9fa2-d04536e253bd.png)

It would be beneficial to security if there was an option to view the creation date of one time passwords in the Settings/Security panel.

**Comment by tcitworld** [comment]
- [ ] Add creation date to `OC\Authentication\Token\IToken` and implementations
- [ ] Expose creation date into settings initialstate
- [ ] Add the column in the VueJS component

**Comment by JeaNugroho** [comment]
After some PHP & Laravel tutorials, Nextcloud server's backend is still far from being intuitive to be understood and worked on. Is the steps mentioned by @tcitworld still the best list of things to follow? If yes, is there a more fine grained list of steps to achieve this enhancement?

**Comment by tcitworld** [comment]
Yup, still the correct way. Improved the message a little to give pointers.
